labs/lab-2-javascript-game/style.css

```diff
@@ -0,0 +1,132 @@
+body {
+  background-color: #000000;
+  padding: 50px;
+}
+
+h1 {
+  font-size: 25pt;
+  font-family: 'VT323', monospace;
+  color: #00ffe1;
+  text-align: center;
+}
+
+h2 {
+  font-size: 15pt;
+  font-family: 'VT323', monospace;
+  color: #00ffe1;
+  text-align: center;
+}
+
+h3 {
+  font-size: 12pt;
+  font-family: 'VT323', monospace;
+  color: #00ffe1;
+  text-align: left;
+}
+
+p {
+  font-size: 12pt;
+  font-family: 'VT323', monospace;
+  color: #FFFFFF;
+  text-align: left;
+}
+
+hr {
+  border-color: #00ffe1;
+}
+
+button {
+  background-color: #00ffe1;
+  color: #000000;
+  width: 200px;
+  margin: 20px;
+  padding: 15px;
+  border: none;
+  font-family: 'VT323', monospace;
+  font-size: 20px;
+}
+
+/* Had trouble getting things exactly in the center, think it has something to do with how the display is set in Java, but couldn't spend any more time figuring it out */
```

Comment:
Note: Java and Javascript are 100% different languages with different purposes, we're learning javascript.

And you wanted to use `justify-content: center`; `align-items: center` centers vertically.